manage.py

```diff
@@ -94,4 +96,12 @@ def prepare_kubernetes_db():
 
 
 if __name__ == "__main__":
+    _host = '127.0.0.1'
```

Comment:
Don't mention the _host and _port hard-coded here.  Rather put it in the default for SERVER_NAME in config.py

Reply:
Once you do that, the entire `if` logic below won't be necessary anymore.